service/src/test/java/uk/gov/hmcts/reform/fpl/Generator.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.fpl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.IOUtils;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.service.DocumentGeneratorService;
+import uk.gov.hmcts.reform.fpl.templates.DocumentTemplates;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseDetails;
+
+public class Generator {
+
+    public static void main(String[] args) throws IOException {
```

Comment:
weve got a `main` method now? is this the main purpose of our service?

Reply:
This class will get removed before we merge to master. James left it for development purpose in case they are some additional changes needed.

Reply:
this still needs removing then